core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1409,3 +1409,15 @@ secret_configs:
       - cluster: app.ci
         namespace: ci
         name: jira-credentials-dptp-bot
+  # This is going to be used by secret/registry-pull-credentials on api.ci only.
+  # The OCP 4 clusters in our build farm have this pull secret already on secret/pull-secret in openshift-config.
+  # We can remove this after deprecating api.ci.
+  # The secret is on app.ci because the applyconfig job for api.ci is there.
+  - from:
+      Pull_Credentials_Token_Only:
```

Comment:
We've never used `Upper_Snake_Case` for anything before - what is wrong with `pull-token` ?